Description: Ignore an additional dependency of the javadoc plugin to work around a build failure
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -662,11 +662,13 @@
 				<artifactId>jakarta.mail</artifactId>
 				<version>${mail.version}</version>
 			    </additionalDependency>
+<!--
 			    <additionalDependency>
 				<groupId>com.sun.mail</groupId>
 				<artifactId>gimap</artifactId>
 				<version>${mail.version}</version>
 			    </additionalDependency>
+-->
 			</additionalDependencies>
 			<sourceFileExcludes>
 			    <sourceFileExclude>
